templates/song_rater/song_detail.html

```diff
@@ -20,5 +20,12 @@ <h2>Rate The Song</h2>
         {% csrf_token %}
       <button type="submit">Vote</button>
     </form>
+    <ul>
+       {% for rating in all_ratings reversed%}
+           <li>Rating:{{ rating.rating }} - {{ rating.date }}</li>
```

Comment:
```suggestion
           <li>Rating: {{ rating.rating }} - {{ rating.date }}</li>
```

I think the list will look nicer this way

Reply:
maybe replace `-` with `at` as well